pkg/cache/cache.go

```diff
@@ -277,8 +301,8 @@ func selectResync(def, override *time.Duration) *time.Duration {
 	return def
 }
 
-func selectNamespace(def, override string) string {
-	if override != "" {
+func selectNamespaces(def, override []string) []string {
+	if len(override) > 0 {
```

Comment:
Do we have to check for nil here instead? Otherwise you can't overwrite with an empty slice. Or is that not a relevant use case?

Reply:
The last time I checked this it was a difference. Let me double-check

Reply:
https://go.dev/play/p/jF2-tgk93OB

Reply:
I _think_ that works as I would expect, the default is a nil slice, which should never overwrite anything. Only slices with at least one member should, so it works like before

Reply:
Yup that's why I asked. Do we ever want to overwrite an existing value with an empty slice (which would then lead to using all namespaces).

But I guess in that case folks can also just overwrite it with a slice with the All value